### PR TITLE
Adds like 17 new logs

### PR DESCRIPTION
**Pull request by moonheart08**
## About the PR 
Adds logs for:
	- Canister breaking, tank insert, tank eject
	- Disarm and disarm knockdown
	- Armed attacks & disarmed attacks
	- Interactions
	- Activations
	- Throwing, throwing hitting shit, and thrown items landing
	- Picking shit up
	- Dropping it again
	- Bullets hitting things (and who shot them)
	- Crayon

:cl: moony
- add: The game admins are now omnipotent.

### CODE REVIEW

File: Content.Server/Atmos/Piping/Unary/EntitySystems/GasCanisterSystem.cs [hunk 1]
```diff
@@ -153,7 +155,7 @@ private void OnCanisterChangeReleaseValve(EntityUid uid, GasCanisterComponent ca
             if (!CheckInteract(args.Session))
                 return;
 
-            _adminLogSystem.Add(LogType.CanisterValve, LogImpact.High, $"{(args.Session as IPlayerSession):player} set the valve on {uid} to {args.Valve:valveState}");
+            _adminLogSystem.Add(LogType.CanisterValve, LogImpact.High, $"{args.Session.AttachedEntity:player} set the valve on {uid} to {args.Valve:valveState}");
```
Comment (juliangiebel) [inline Content.Server/Atmos/Piping/Unary/EntitySystems/GasCanisterSystem.cs:158]: Wouldn't the impact depend on whether there is a tank inside the canister or not?

Comment (moonheart08) [reply Content.Server/Atmos/Piping/Unary/EntitySystems/GasCanisterSystem.cs:158]: Good point, sec

File: Content.Server/Atmos/Piping/Unary/EntitySystems/GasCanisterSystem.cs [hunk 2]
```diff
@@ -275,6 +277,9 @@ private void OnCanisterInteractUsing(EntityUid uid, GasCanisterComponent compone
             if (!hands.Drop(args.Used, container))
                 return;
 
+            _adminLogSystem.Add(LogType.CanisterTankInserted, LogImpact.Medium, $"Player {args.User:player} inserted tank {container.ContainedEntities[0]} into {uid}");
+
+
```
Comment (juliangiebel) [inline Content.Server/Atmos/Piping/Unary/EntitySystems/GasCanisterSystem.cs:287]: Whitespace

**Review by juliangiebel — APPROVED** [review]
lgtm

**Comment by moonheart08** [comment]
what even is that test failure, wtf

File: Content.Server/Atmos/Piping/Unary/EntitySystems/GasCanisterSystem.cs [hunk 3]
```diff
@@ -66,6 +66,7 @@ public void PurgeContents(EntityUid uid, GasCanisterComponent? canister = null,
             if (environment is not null)
                 _atmosphereSystem.Merge(environment, canister.Air);
 
+            _adminLogSystem.Add(LogType.CanisterPurged, LogImpact.Medium, $"Canister {uid} purged it's contents of {canister.Air} into the environment.");
```
Comment (PJB3005) [inline Content.Server/Atmos/Piping/Unary/EntitySystems/GasCanisterSystem.cs:69]:
```suggestion
            _adminLogSystem.Add(LogType.CanisterPurged, LogImpact.Medium, $"Canister {uid} purged its contents of {canister.Air} into the environment.");
```

**Pull request merged**